### PR TITLE
copy_expert() added to execute query COPY FROM STDIN

### PR DESCRIPTION
**Pull request by brahim-cher**

### CODE REVIEW

File: gobcore/datastore/postgres.py [hunk 1]
```diff
@@ -92,6 +92,19 @@ def execute(self, query: str) -> None:
         except psycopg2.Error as e:
             raise GOBException(f'Error executing query: {query[:80]}. Error: {e}')
 
+    def copy_expert(self, query, data) -> None:
```
Comment (roel-devries) [inline gobcore/datastore/postgres.py:95]: add some typing here for the parameters

File: gobcore/datastore/sql.py [hunk 2]
```diff
@@ -27,6 +27,10 @@ def write_rows(self, table: str, rows: List[list]) -> None:
     def execute(self, query: str) -> None:
         pass  # pragma: no cover
 
+    @abstractmethod
+    def copy_expert(self, query: str, data: str) -> None:
```
Comment (roel-devries) [inline gobcore/datastore/sql.py:31]: I think copy_export is specific for postgresql?

File: gobcore/datastore/postgres.py (hunk 1 above)
Comment (brahim-cher) [reply gobcore/datastore/postgres.py]: Done.

File: gobcore/datastore/sql.py (hunk 2 above)
Comment (brahim-cher) [reply gobcore/datastore/sql.py:31]: I think so! The copy_expert(query, data) used is imported from psycopg2 lib as the execute() function.

Comment (roel-devries) [reply gobcore/datastore/sql.py:31]: yes, so the abstractmethod can be removed here ;) (oracle is for instance also a subclass)

Comment (brahim-cher) [reply gobcore/datastore/sql.py]: een goede! Done.

**Review by roel-devries — DISMISSED** [review]

**Review by roel-devries — APPROVED** [review]

**Pull request merged**